tests/integration_tests/test_qwen_embedding_function.py

```
@@ -0,0 +1,215 @@
+    reason="QWEN_API_KEY environment variable must be set"
+)
+class TestQwenEmbeddingFunction:
+    """Test QwenEmbeddingFunction - skipped by default, requires manual execution"""
+
+    def test_litellm_env(self):
+        """Test if litellm is installed and required environment variables are set."""
+        try:
+            import litellm
+        except ImportError:
+            print("litellm package is not installed")
+            assert False, "litellm package is not installed"
```

Ruff (0.14.10)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)